core/src/test/java/me/uport/sdk/core/NetworksTest.kt

```diff
@@ -39,6 +39,36 @@ class NetworksTest {
             .isEqualTo(Networks.mainnet.ethrDidRegistry)
     }
 
+    @Test
+    fun `can find a registered network using the name`() {
+
+        assertThat(Networks.get("mainnet")).isEqualTo(Networks.mainnet)
+
+        assertThat(Networks.get("rinkeby")).isEqualTo(Networks.rinkeby)
+
+        assertThat(Networks.get("kovan")).isEqualTo(Networks.kovan)
+
+        assertThat(Networks.get("ropsten")).isEqualTo(Networks.ropsten)
+    }
+
+    @Test
+    fun `throws error if the network name does not match`() {
+        assertThat {
+            Networks.get("manet")
+        }.thrownError {
+            isInstanceOf(IllegalStateException::class)
+        }
+    }
+
+    @Test
+    fun `throws error if the network name cases does not match`() {
+        assertThat {
+            Networks.get("Mainnet")
```

Comment:
I think this test should actually pass.
Network names (and I think it should also apply to `networkId`s) should not be case-sensitive